app/javascript/components/mentoring/Queue/Solution.jsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react'
```

Comment:
There is now a double React import which leads to a runtime error. Could you remove the first line?

```suggestion
```